notes/000-initial-commit.md

```diff
@@ -1,11 +1,6 @@
 The one where the initial commit preparations are described
 ===
 
-So, here's the start of the project. It's always good to get something working first and build on top of that, so the work starts from the backend. And the backend starts with choosing appropriate tech and tooling:
-* Version control system - [Git](https://git-scm.com) looks like an obvious choice for a layman, because it's so widespread and there are well-known platforms for code hosting, like GitHub.
-* Backend tech - [ASP.NET Core](https://www.microsoft.com/net/core) is a solid candidate. .NET ecosystem is very mature, so the lack of documentation or features won't be a probem, and it's cross-platform and familiar for me as well.
```

Comment:
(nit) I would leave text about .NET Core and VS Code as it is or as another option in simplified way